docs/apache-airflow-providers-tabular/index.rst

```diff
@@ -0,0 +1,77 @@
+
+ .. Licensed to the Apache Software Foundation (ASF) under one
+    or more contributor license agreements.  See the NOTICE file
+    distributed with this work for additional information
+    regarding copyright ownership.  The ASF licenses this file
+    to you under the Apache License, Version 2.0 (the
+    "License"); you may not use this file except in compliance
+    with the License.  You may obtain a copy of the License at
+
+ ..   http://www.apache.org/licenses/LICENSE-2.0
+
+ .. Unless required by applicable law or agreed to in writing,
+    software distributed under the License is distributed on an
+    "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+    KIND, either express or implied.  See the License for the
+    specific language governing permissions and limitations
+    under the License.
+
+``apache-airflow-providers-tabular``
+====================================
+
+Content
+-------
+
+.. toctree::
+    :maxdepth: 1
+    :caption: Guides
+
+    Connection types <connections>
+
+.. toctree::
+    :hidden:
+    :caption: System tests
+
+    System Tests <_api/tests/system/providers/tabular/index>
+
+.. toctree::
+    :maxdepth: 1
+    :caption: Resources
+
+    Example DAGs <https://github.com/apache/airflow/tree/providers-tabular/1.0.0/tests/system/providers/tabular>
+    PyPI Repository <https://pypi.org/project/apache-airflow-providers-tabular/>
+    Installing from sources <installing-providers-from-sources>
+    Python API <_api/airflow/providers/tabular/index>
+
+
+.. THE REMAINDER OF THE FILE IS AUTOMATICALLY GENERATED. IT WILL BE OVERWRITTEN AT RELEASE TIME!
+
+
+.. toctree::
+    :maxdepth: 1
+    :caption: Commits
+
+    Detailed list of commits <commits>
+
+
+Package apache-airflow-providers-tabular
+------------------------------------------------------
+
+`Tabular <https://tabular.io/>`__
+
+
+Release: 0.0.1
+
+Provider package
+----------------
+
+This is a provider package for ``tabular`` provider. All classes for this provider package
+are in ``airflow.providers.tabular`` python package.
+
+Installation
+------------
+
+You can install this package on top of an existing Airflow 2.1+ installation via
```

Comment:
```suggestion
You can install this package on top of an existing Airflow 2.2+ installation via
```
The minimum version for providers was bumped to 2.2.0 recently 🙂

Reply:
Missed that one, I'll do a follow-up! 👍🏻